### PR TITLE
Gen3 Monthly Release 2020.08 nci-crdc-staging.datacommons.io 1595902131

### PR DESCRIPTION
**Pull request by themarcelor**
Applying version 2020.08 to nci-crdc-staging.datacommons.io

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# nci-crdc-staging.datacommons.io/manifest.json
## Deployment changes
- portal
  - Portal sould be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)

**Comment by PlanXCyborg** [comment]
# nci-crdc-staging.datacommons.io/manifest.json
## Deployment changes
- portal
  - Portal sould be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)

**Comment by PlanXCyborg** [comment]
# nci-crdc-staging.datacommons.io/manifest.json
## Deployment changes
- fence
  - If the fence config already has [CLIENT_, USER_, SESSION_]ALLOWED_SCOPES fields, and GA4GH AAI integration is desired, then add ga4gh_passport_v1 to all those lists. If default config values are being used then no change required. (uc-cdis/uc-cdis/fence#799)
- portal
  - Portal should be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)

**Comment by PlanXCyborg** [comment]
# nci-crdc-staging.datacommons.io/manifest.json
## Deployment changes
- fence
  - If the fence config already has [CLIENT_, USER_, SESSION_]ALLOWED_SCOPES fields, and GA4GH AAI integration is desired, then add ga4gh_passport_v1 to all those lists. If default config values are being used then no change required. (uc-cdis/uc-cdis/fence#799)
- portal
  - Portal should be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)

File: nci-crdc-staging.datacommons.io/manifests/modsec/crawlers-user-agents.data [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Search engine crawlers and other bots
```
Comment (jiaqi216) [inline nci-crdc-staging.datacommons.io/manifests/modsec/crawlers-user-agents.data:1]: do we need the modsec stuff?

Comment (themarcelor) [reply nci-crdc-staging.datacommons.io/manifests/modsec/crawlers-user-agents.data]: Not right now.., removing it.

**Comment by PlanXCyborg** [comment]
# nci-crdc-staging.datacommons.io/manifest.json
## Deployment changes
- fence
  - If the fence config already has [CLIENT_, USER_, SESSION_]ALLOWED_SCOPES fields, and GA4GH AAI integration is desired, then add ga4gh_passport_v1 to all those lists. If default config values are being used then no change required. (uc-cdis/uc-cdis/fence#799)
- portal
  - Portal should be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)

**Pull request merged**